tools/opensnoop_example.txt

```diff
@@ -156,30 +156,31 @@ to the '-n' option.
 The -e option prints out extra columns; for example, the following output
 contains the flags passed to open(2), in octal:
 
-# ./opensnoop -e
-PID    COMM               FD ERR FLAGS    PATH
-28512  sshd               10   0 00101101 /proc/self/oom_score_adj
-28512  sshd                3   0 02100000 /etc/ld.so.cache
-28512  sshd                3   0 02100000 /lib/x86_64-linux-gnu/libwrap.so.0
-28512  sshd                3   0 02100000 /lib/x86_64-linux-gnu/libaudit.so.1
-28512  sshd                3   0 02100000 /lib/x86_64-linux-gnu/libpam.so.0
-28512  sshd                3   0 02100000 /lib/x86_64-linux-gnu/libselinux.so.1
-28512  sshd                3   0 02100000 /lib/x86_64-linux-gnu/libsystemd.so.0
-28512  sshd                3   0 02100000 /usr/lib/x86_64-linux-gnu/libcrypto.so.1.0.2
-28512  sshd                3   0 02100000 /lib/x86_64-linux-gnu/libutil.so.1
+# ./opensnoop.py -e
+PID    COMM               FD ERR FLAGS    MODE PATH
+312769 cgroupify           5   0 02204000 0000 .
+312769 cgroupify           5   0 00000000 0000 312814/cgroup.procs
```

Comment:
The value 0000 for MODE has different meanings depending on the flag value. 
In the case where a file is created, 0000 has a valid meaning, but when it is not a file creation case, 0000 represents the default value of the mode variable.
How about using a distinct expression (such as a blank space or "n/a") instead of 0000 for clarity and user convenience?

Reply:
Thanks, you're right, just fix it like:

```diff
+# ./opensnoop.py -e
+PID    COMM               FD ERR FLAGS    MODE PATH
+12458  open                3   0 02000000 n/a  /etc/ld.so.cache
+12458  open                3   0 02000000 n/a  /lib/x86_64-linux-gnu/libc.so.6
+12458  open                3   0 00000301 0664 tmp.txt
```